src/components/Inputs/AudioInput/AudioInput.vue

```diff
@@ -0,0 +1,385 @@
+      <template v-if="audio.state === 'paused'">
+        <div class="play-pause-button">
+          <PbIcon
+            :icon="playerIcon"
+            size="fa-xs"
+            class="pb-audio-player-icon"
+            @click.native="togglePlay"
+          />
+        </div>
+
+        <div class="pb-timeline-player">
+          <div class="lines">
+            <div
+              class="progress"
+              :style="`width: ${layerX}px`"
+            />
+
+            <div
+              class="remaining"
+              :style="`width: ${audio.width - layerX}px`"
+            />
+          </div>
+
+          <div
+            class="ball"
+            :style="`margin-left: ${layerX}px`"
+            @drag="dragTimer"
+          />
+        </div>
+
+        <div class="duration">
+          <p class="pb">{{ formattedDuration }}</p>
+        </div>
+      </template>
+
+      <template v-else>
+        <div class="recording-ball">
+          <PbIcon
+            icon="fas fa-circle"
+            style="color: var(--color-danger); width: 8px;"
+          />
+        </div>
+        <div class="timeline">
+          <hr>
+        </div>
+        <div class="duration">
+          <p class="pb">{{ formattedDuration }}</p>
+        </div>
+      </template>
+    </div>
+
+    <PbIcon
+      :icon="rightIcon"
+      class="pb-audio-change-state-icon"
+      size="fa-md"
+      @click="toggleRecorder"
+    />
+  </div>
+</template>
+
+<script>
+import PbIcon from '../../Miscellaneous/Icon/Icon';
+
+export default {
+  name: 'PbAudioInput',
+
+  components: {
+    PbIcon,
+  },
+
+  props: {
+    startRecording: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+  emits: ['change-state', 'audio'],
+
+  data() {
+    return {
+      mediaRecorder: null,
+      audio: {
+        chunks: [],
+        recorded: null,
+        state: 'paused',
+        width: 200,
+        element: null,
+        duration: 0,
+        timeline: 0,
+        layerX: 0,
+        isPlaying: false,
+        interval: null,
+      },
+    };
+  },
+
+  computed: {
+    rightIcon() {
+      return this.audio.state === 'paused' ? 'fas fa-microphone' : 'fas fa-pause';
+    },
+
+    playerIcon() {
+      return this.audio.isPlaying ? 'fas fa-pause' : 'fas fa-play';
+    },
+
+    duration() {
+      return this.audio.element?.duration !== Infinity && !!this.audio.element?.duration
+        ? this.audio.element?.duration
+        : this.audio.duration;
+    },
+
+    formattedDuration() {
+      const minutes = this.duration > 60 ? Math.floor(this.duration / 60) : 0;
+      const seconds = this.duration > 60
+        ? String(Math.floor(this.duration - minutes * 60)).padStart(2, '0')
+        : String(Math.floor(this.duration)).padStart(2, '0');
+
+      return `${minutes}:${seconds}`;
+    },
+
+    layerX() {
+      return this.audio.timeline;
+    },
+  },
+
+  mounted() {
+    if (this.startRecording) this.toggleRecorder();
+  },
+
+  breforeDestroy() {
```

Comment:
<img src="https://media2.giphy.com/media/alvlhqPcBSNDs4wE3P/giphy.gif"/>

Reply:
https://www.figma.com/file/EvuzOS2hbbaU4MdUwzKR1k/Atendimento-S%2B?type=design&node-id=2946-148392&t=izbvMnFmCdGrdUKp-0
As in the flow of record a audio on customer-service, we will click a external button to start recording. So the component needs to start recording when it mounts.
This is why of this prop. The prop name is changed now... for better understanding